specify/specify7
# fix simple search QueryField constructor call

Fixes #5425 

Fixes the errors from occurring during any simple search by fixing the constructor call to QueryField in the simple search code.

### Checklist

      and self-explanatory (or properly documented)

### Testing instructions

1. Try running any simple search to make sure that no toats appear with error messages related to a TypeError
